lib/Controller/AEnvironmentAwareController.php

```diff
@@ -59,4 +60,24 @@ public function setParticipant(Participant $participant): void {
 	public function getParticipant(): ?Participant {
 		return $this->participant;
 	}
+
+	/**
+	 * @return string Either 'json' or 'xml'
+	 */
+	public function getResponseFormat(): string {
+		// get format from the url format or request format parameter
+		$format = $this->request->getParam('format');
```

Comment:
If in format have blablabla will pass, not?!
Isn't good to check if format is in a list of allowed formats?

Reply:
Server also does not care:
https://github.com/nextcloud/server/blob/368f83095d443c28b0d90d2308549349cc28dd05/lib/private/AppFramework/Http/Dispatcher.php#L231-L244

Will add that link as a comment

Reply:
In the server maybe make sense because the API of apps could deliver specifics formats different from json and xml, but in this case only is valid json and xml.